presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -232,7 +232,17 @@ Property Name                                                         Descriptio
 ``hive.invalidate-metastore-cache-procedure-enabled``    When enabled, users will be able to invalidate metastore        false
                                                          cache on demand.
 
-======================================================= ============================================================= ============
+``hive.metastore.thrift.client.tls.enabled``             Whether TLS security is enabled.                                false
```

Comment:
The release notes shall show these property names. E.g. "Add a configuration property plan-checker.config-dir to set the configuration directory for PlanCheckerProvider configurations. [#23955](https://github.com/prestodb/presto/pull/23955)"